src/Nox.Types/Types/Color/Color.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Color"/> type and value object. 
+/// </summary>
+public class Color : ValueObject<byte[], Color>
+{
+    private byte _alpha { get; set; }
+    private byte _red { get; set; }
+    private byte _green { get; set; }
+    private byte _blue { get; set; }
+    
+    public Color()
+    {
+        Value = new byte[4];
+    }
+
+    private bool _isEmpty { get; set; }
+
+    public static Color Empty
+    {
+        get => new() { _isEmpty = true };
+    }
+
+    /// <summary>
+    /// Validates a <see cref="Color"/> object.
+    /// </summary>
+    /// <returns>true if the <see cref="Color"/> value is valid.</returns>
+    internal override ValidationResult Validate()
+    {
+        var result = new ValidationResult();
+
+        if (Value.Length < 1)
+            result.Errors.Add(new ValidationFailure(nameof(Value),
+                $"Could not create a Nox {nameof(Color)} type as an empty {nameof(Value)} is not allowed."));
+
+        if (Value.Length > 4)
+            result.Errors.Add(new ValidationFailure(nameof(Value),
+                $"Could not create a Nox {nameof(Color)} type with more than 4 values."));
+
+        return result;
+    }
+
+    public static Color From(byte red, byte green, byte blue)
+        => From(new byte[] { 0, red, green, blue });
```

Comment:
I think we should set Alpha for the maximum value. It doesn't make sense for it to return me a transparent (no color) when I'm creating a colors by sending its RGB values.
This is a just a suggestion since it's more of a design choice.

Reply:
I think it could be ok, because 255 in alpha should be fully opaque.